### ISSUE TITLE
Bug in `k_ring_distances()`

### ISSUE THREAD

**Issue by kwmsmith** [uber/h3-py#42]
In the vicinity of the pentagon `870800000ffffff`:

```python
>>> h3.k_ring_distances('870800003ffffff', 2)
[{'0', '870800003ffffff'},
 {'870800000ffffff',
  ...},
 {'870800004ffffff',
  ...}]
```

Note the extra null hex in the 0th set above.

Doesn't show up in `h3.k_ring('870800003ffffff', 2)` or when `k < 2`.

**Comment by nrabinowitz** [comment]
This is expected output from the C core; the bindings should be responsible for removing nulls (or I suppose explicitly passing this responsibility to their consumers).

**Comment by kwmsmith** [comment]
Right, in all other API functions the python wrapper removes nulls, looks like null pruning was missed for this case.

**Comment by karentycoon** [comment]
@kwmsmith Thanks for reporting! We just released version 3.4.3, and this bug is fixed in the new version.
